databaseAnnotator/src/main/java/org/cbioportal/database/annotator/AnnotateRecordsProcessor.java

```diff
@@ -114,13 +114,6 @@ else if (annotatedRecord.getVARIANT_CLASSIFICATION().equals("5'Flank") ||
         String chr = !(annotatedRecord.getCHROMOSOME() == null || annotatedRecord.getCHROMOSOME().isEmpty()) ? annotatedRecord.getCHROMOSOME() : i.getCHR();
         Integer start = i.getSTART_POSITION();
         Integer end = i.getEND_POSITION();
-        try {
```

Comment:
Mutation events should never have null or NA values but maybe this should be added back in case we accidentally get a fusion event when querying for mutation events with `protein_change=MUTATED`.